Services/user/Api/Controllers/order.js

```diff
@@ -0,0 +1,55 @@
+const { Order, Product } = require('../../database/models/');
+
+class OrderController {
+  async createOrder(call, callback) {
+    try {
+      const newOrder = await Order.create(call.request);
+      callback(null, newOrder.dataValues.id);
+    } catch (error) {
+      callback(error);
+    }
+  }
+  async getOrder(call, callback) {
+    try {
+      const getOne = await Order.findById(call.request.id);
+      const payload = {
+        id: getOne.dataValues.id,
+        payed: getOne.dataValues.payed,
+        userId: getOne.dataValues.user,
+        totally: getOne.dataValues.totally || -1,
+      };
+      callback(null, payload);
+    } catch (error) {
+      callback(error);
+    }
+  }
+  async addProduct(call, callback) {
+    try {
+      const getOneOrder = await Order.findById(call.request.idOrder);
+      if (getOneOrder.dataValues.id === call.request.idOrder) {
```

Comment:
Redundant?

Reply:
I couldn't remember why I did it, but currently is fix